packages/scripts/create-version.sh

```diff
@@ -1,86 +1,137 @@
 #!/usr/bin/env bash
 set -e
 
-#===================================================================================
-# Usage:
-#   createVersion <major|minor|patch>
-#
-# Description:
-#   Create new version of package to be published on NPM.
-#   Before pushing changes to remote, build and test npm scripts are executed.
-#
-# Globals:
-#   MAIN_BRANCH - (optional) Repository main branch (default: master)
-#
-# Arguments:
-#   $1 (version type) - "patch", "minor" and "major" strings allowed
-#
-# Returns:
-#   None
-#===================================================================================
+help()
+{
+   echo
+   echo "Creates new version of a package to be published on NPM."
+   echo "Before pushing changes to remote the following NPM scripts - if present - are executed:"
+   echo "build, lint, check-types, test."
+   echo
+   echo "Syntax:"
+   echo "  create-version.sh <version-type> [<package-name>]"
+   echo
+   echo "Environment variables:"
+   echo "  MAIN_BRANCH - (optional) Repository main branch (default: master)"
+   echo
+   echo "Arguments:"
+   echo "  version-type - version type supported by NPM version script"
+   echo "                 examples: patch, minor, major, prepatch, preminor, premajor, prerelease,"
+   echo "                 see https://docs.npmjs.com/cli/v8/commands/npm-version for details"
+   echo "  package-name - package directory name, relevant only for cloudify-ui-common repository,"
+   echo "                 examples: backend, configs, cypress, frontend, scripts"
+   false
+}
 
-createVersion() {
-    log() {
-        LOG_PREFIX="create-version.sh:"
-        echo -e $LOG_PREFIX $@
-    }
 
-    NPM_PACKAGE_NAME=`cut -d "=" -f 2 <<< $(npm run env | grep "npm_package_name")`
+log() {
+    LOG_PREFIX="create-version.sh:"
+    echo -e $LOG_PREFIX $@
+}
 
-    MAIN_BRANCH=${MAIN_BRANCH:-master}
-    log "Switching to main branch (${MAIN_BRANCH}) and checking if its up-to-date...";
-    git checkout ${MAIN_BRANCH} || {
-      log "Error during checking out ${MAIN_BRANCH} branch"
-      false
-    }
-    git fetch || {
-      log "Error during fetching data from repository"
-      false
-    }
-    git pull || {
-      log "Error during pulling ${MAIN_BRANCH} branch"
-      false
-    }
-    git diff origin/${MAIN_BRANCH} --exit-code --compact-summary || {
-      log "Working directory not clean. Delete or push your changes to ${MAIN_BRANCH}}."
-      false
-    }
+error() {
+    log "ERROR: " $@
+}
 
-    log "Running tests and build..."
-    npm run test || {
-      log "Running tests failed"
-      false
-    }
-    npm run build || {
-      log "Running build failed"
-      false
-    }
+VERSION_TYPE=$1
+PACKAGE_NAME=$2
 
-    NEXT_VERSION=$(npm version --no-git-tag-version "$@")
```

Comment:
`npm version` works differently for workspaces (not producing single line output with version), so I couldn't use it in all cases.